errored.go

```diff
@@ -152,11 +152,13 @@ func (e *Error) Error() string {
 	}
 
 	if e.trace || AlwaysTrace {
-		ret := desc + "\n"
-
-		for _, stack := range e.stack {
-			for _, line := range stack {
-				ret += fmt.Sprintf("\t%s [%s %d]\n", line.fun, line.file, line.line)
+		var ret string
+		for i := 0; i < len(e.errors); i++ {
+			ret += fmt.Sprintf("%s\n", e.errors[i].Error())
+			if len(e.stack) > i {
```

Comment:
@erikh 

> We even get a stack trace at Combine time.

I am not sure if this is true in the current code for **non-Errored** errors from line: https://github.com/contiv/errored/pull/11/files/180ad0e9df6ff1fe3a629add66e0f69d9938a66c#diff-57f00f2899a684aeeac51afa14c217e2L140

I quickly tried a dummy test in this commit (https://github.com/contiv/errored/commit/b5057a2e4ccbce1866a8c8c1087fed8c1709001f) to verify my understanding that I shared above and this is what I see. Note that the stack for errored error `bar` get's incorrectly reported against non-errored error `foobar`. Also the panic is the result of a mismatch between number of combined errors and the number of combined stack traces:

```
--- FAIL: TestErrorCombined (0.00s)
        errored_test.go:138: mismatch between combined errors and combined stacks
        errored_test.go:141: foo: [{errored_test.go 136 github.com/contiv/errored.TestErrorCombined} {testing.go 473 testing.tRunner} {asm_amd64.s 1998 runtime.goexit}]
        errored_test.go:141: foobar: [{errored_test.go 136 github.com/contiv/errored.TestErrorCombined} {testing.go 473 testing.tRunner} {asm_amd64.s 1998 runtime.goexit}]
panic: runtime error: index out of range [recovered]
        panic: runtime error: index out of range
```